packages/solid/components/Row/Row.tsx

```diff
@@ -26,7 +26,17 @@ import type { RowProps } from './Row.types.js';
 const Row: Component<RowProps> = (props: RowProps) => {
   const onLeft = handleNavigation('left');
   const onRight = handleNavigation('right');
-  const scroll = createMemo(() => withScrolling(true, props.x));
+  const scroll = createMemo(() => {
+    let x;
+    if (props.style) {
+      if (Array.isArray(props.style)) {
+        x = props.style[0].x || 0;
+      } else {
+        x = props.style.x || 0;
+      }
+    }
+    return withScrolling(false, props.x || x);
+  });
```

Comment:
same as the Column:

in an instance where the props.style[0].x is undefined but props.style[1].x is, this will set y to 0 instead of the correct value. is there a way we can write this to be a little more resilient?